packages/opentelemetry-tracer-web/examples/js/index.js

```diff
@@ -0,0 +1,34 @@
+import { WebTracer } from '../../src';
+
+import * as shimmer from 'shimmer';
```

Comment:
Although I think use of shimmer is cool/useful in automatic tracing contexts, would it make sense for the example to be a little more basic and just show how to e.g. create a span for some code that does a timeout or XHR call?

Reply:
I have used it as some playground to debug stuff and understand this better :). Wasn't really sure what to show as an example and how many of them we should have.  Maybe we can talk a bit more to have similar examples in other places too?

Reply:
Yeah, I think this is OK for now. In the future, we can always improve the examples in various ways.